.github/workflows/build.yml

```diff
@@ -274,6 +274,33 @@ jobs:
             ${{ github.workspace }}/packages/**/*.tgz
             ${{ github.workspace }}/packages/serverless/dist-awslambda-layer/*.zip
 
+  job_playwright_test:
+    name: Playwright E2E Tests
+    needs: job_build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out current commit (${{ github.sha }})
+        uses: actions/checkout@v2
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
```

Comment:
Why node `v12`?

Reply:
Playwright currently requires node 12+: https://github.com/microsoft/playwright/issues/2059#issuecomment-622128167

Reply:
We can probably use the latest version of Node here, since the runner environment != the browser targets we're interested in testing.